alert-database/src/main/java/com/synopsys/integration/alert/database/api/DefaultUserAccessor.java

```diff
@@ -121,7 +113,7 @@ public UserModel updateUser(UserModel user, boolean passwordEncoded) throws Aler
         Long existingUserId = existingUser.getId();
         UserEntity savedEntity = existingUser;
         // if it isn't an external user then update username, password, and email.
-        if (existingUser.isExternal()) {
+        if (!existingUser.getAuthenticationType().equals("DATABASE")) {
```

Comment:
We may want to change this to `AuthenticationType.DATABASE.name().equals(existingUser.getAuthenticationType())`. This way if we change DATABASE we only have to change it in one place.